.github/workflows/cd.yml

```diff
@@ -138,15 +138,23 @@ jobs:
           npx lerna publish from-package --yes
 
       - name: Install Mustache
-        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') }}
         run: curl -sSL https://git.io/get-mo -o mo && chmod +x mo && sudo mv mo /usr/local/bin/
 
+      - name: sync up templates dependency on main release
+        if: ${{ !contains(steps.version-change.outputs.CHANGED, 'templates@') && contains(steps.version-change.outputs.CHANGED, '@microsoft/teamsfx') }}
```

Comment:
I think even `contains(steps.version-change.outputs.CHANGED, 'templates@'` is true, we should also sync it.

Reply:
synced offline

Reply:
if sdk version changed, and template has no change, so we need to check templates deps weather need to sync up with sdk version